packages/plugins/embarkjs/src/index.js

```diff
@@ -1,4 +1,5 @@
 import {__} from 'embark-i18n';
+const Web3 = require('web3');
```

Comment:
Would prefer to see:

```suggestion
import Web3 from 'web3';
```